build/parseSpec.c

```diff
@@ -533,7 +534,17 @@ int readLine(rpmSpec spec, int strip)
 	}
 	*endFileName = '\0';
 
+	/* System includes live in a special path and become buildrequires */
+	if (*fileName == '<' && *(endFileName-1) == '>') {
+	    *(endFileName-1) = '\0';
+	    sysinc = rpmGetPath("%{_rpmincludedir}/", fileName + 1, NULL);
+	    addReqProv(spec->sourcePackage, RPMTAG_REQUIRENAME,
+			sysinc, NULL, 0, 0);
```

Comment:
I'm not sure about this one, because you are technically do not depend on exact `/usr/lib/rpm/include/…` file, but rather that path will be present...

Reply:
The point is that other tooling knows how to look at buildrequires, but they dont (and should not) try to parse rpmbuild output for missing files.